src/backends/postgresql/blob.cpp

```diff
@@ -22,6 +23,20 @@
 using namespace soci;
 using namespace soci::details;
 
+// We need this helper function when using ancient PostgreSQL versions without
+// support for 64-bit offets.
+#if PG_VERSION_NUM < 90300
+static int pg_check_fits_32_bits(std::size_t len)
+{
+    if (len > 0x7fffffff)
```

Comment:
This would be a tad cleaner by using `std::numeric_limits<std::uint32_t>::max()` instead of `0x7fffffff`

Reply:
I admit I was lazy, but we'd need to include `<cstdint>` then and then remember to remove it when we finally drop support for Postgres 9.2, so I thought it would be simpler to just do it like this.